src/libraries/Common/src/System/Net/Internals/IPEndPointExtensions.cs

```diff
@@ -39,6 +39,10 @@ public static EndPoint Create(this EndPoint thisObj, Internals.SocketAddress soc
 
                 return socketAddress.GetIPEndPoint();
             }
+            else if (family == AddressFamily.Unknown)
```

Comment:
What's the purpose of this method? Returning the same instance is ok?

Reply:
I was not sure. It seem like the socket is trying to convert from SocketAddress to Internal.SocketAddress using PAL. Since there is no pal for unknown AF, I feel giving back the same is ok (and it does work). If not I can create copy.